### PR TITLE
Suggested change for issue #23, Hybrid Types section expansion

### PR DESCRIPTION
**Pull request by starksm64**

### CODE REVIEW

**Comment by mhegazy** [comment]
:+1:

File: pages/Interfaces.md [hunk 1]
```diff
@@ -273,10 +273,17 @@ interface Counter {
     reset(): void;
 }
 
-var c: Counter;
+function getCounter():Counter{
```
Comment (DanielRosenwasser) [inline pages/Interfaces.md]: Space after `:`

File: pages/Interfaces.md [hunk 2]
```diff
@@ -273,10 +273,17 @@ interface Counter {
...
 
-var c: Counter;
+function getCounter():Counter{
+    var counter = <Counter>function(start:number){};
+    counter.interval = 123;
+    counter.reset = function(){};
+    return counter;
+}
+
+var c: Counter = getCounter();
```
Comment (DanielRosenwasser) [inline pages/Interfaces.md]: Omit the type annotation.

**Comment by mhegazy** [comment]
closing in favor of  #107

**Pull request closed without merging**